tests/login_test.py

```diff
@@ -58,7 +62,7 @@ async def run():
                     device_secret=cloud_client._mqtt_credentials['deviceSecret'], iot_token=cloud_client._iotCredentials['iotToken'], client_id=cloud_client._client_id)
 
     luba._cloud_client = cloud_client
-    # luba.connect() blocks further calls
+    #luba.connect() blocks further calls
```

Comment:
```suggestion
    # luba.connect() blocks further calls
```
commented out code or statements need a space.